pkg/mock/fileUtils.go

```diff
@@ -166,21 +204,21 @@ func (f *FilesMock) FileRemove(path string) error {
 		return fmt.Errorf("the file '%s' does not exist: %w", path, os.ErrNotExist)
 	}
 	absPath := f.toAbsPath(path)
-	content, exists := f.files[absPath]
+	_, exists := f.files[absPath]
 	if !exists {
 		return fmt.Errorf("the file '%s' does not exist: %w", path, os.ErrNotExist)
 	}
 	delete(f.files, absPath)
-	f.removedFiles[absPath] = content
+	f.removedFiles[absPath] = nil
```

Comment:
Is this because you want to deliberately remove the possibility to check the content (and/or permissions) of removed files? Otherwise you could just transfer the props pointer.